packages/hooks/feed/useSocialReactions.ts

```diff
@@ -61,12 +77,13 @@ export const useSocialReactions = ({
       client,
       msg: {
         icon: emoji,
-        identifier: post.localIdentifier,
+        identifier: "post.localIdentifier",
```

Comment:
do you mean `${post.localIdentifier}` ?

Reply:
Dummy mistake, my bad...
https://github.com/TERITORI/teritori-dapp/pull/1393/commits/0db0ab22f2a177e470025608c3f2948e28016a8e